src/Model.elm

```diff
@@ -98,6 +102,20 @@ update msg model =
         CreateFail err ->
             ( { model | error = True, errorMsg = (toString err) }, Cmd.none )
 
+        DeleteRecord recordId ->
+            ( model, deleteRecord recordId )
+
+        DeleteRecordSucceed { data } ->
+            ( { model
+                | records = removeRecordFromList data model.records
+                , error = False
+              }
+            , Cmd.none
```

Comment:
Excellent question. We should take some time to see if we want to "experiment" with optimistic updates or not, and if we want to try optimizing (removing an item from the list) or not (refreshing the whole list).